src/promptflow-devkit/promptflow/_sdk/_user_agent.py

```diff
@@ -1,6 +1,6 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
-from promptflow._version import VERSION
+from promptflow.client._version import VERSION
 
 USER_AGENT = "{}/{}".format("promptflow-sdk", VERSION)
```

Comment:
how about copy the code here: 
![image](https://github.com/microsoft/promptflow/assets/47586720/a68b28cf-bb16-4d30-b90a-d25a1ec32ac3)

So we can avoid import things from promptflow.client and causing circular import.